docs/features.md

```diff
@@ -209,10 +209,75 @@ Make Julia start listening on a given port and return lint messages to requests
 This feature is useful when you want to lint julia code in a non julia environment.
 
 Existing plugins:
+
  * Sublime Text: [SublimeLinter-contrib-julialintserver](https://github.com/invenia/SublimeLinter-contrib-julialintserver)
  * linter-julia for Atom: [linter-julia](https://github.com/TeroFrondelius/linter-julia)
 
-The protocol for the server is:
+The new protocol for the server is JSON in both input and output:
+```json
+"{
+    \"file\":\"path_to_the_file\",
+    \"code_str\":\"full_text_of_the_file\",
+    \"ignore_codes\":[\"E381\",\"W361\",\"I171\"],
+    \"ignore_info\":false,
+    \"ignore_warnings\":false,
+    \"show_code\":true
+}"
+```
+Only the two first `'file'` and `'code_str'` are mandatory fields. For the output
+there are four different protocols from which the `"lint-message"` is preferred,
+because it is the direct match of `LintMessage` and this way will be always up
```

Comment:
I wouldn't say it is preferred, because the `LintMessage` format, while up to date, is subject to change.